docker/dist/Dockerfile.dist.centos7

```diff
@@ -17,21 +17,17 @@
 
 FROM centos:centos7
 
-RUN yum -y upgrade
-RUN yum -y install python; yum clean all
-RUN yum -y install unzip; yum clean all
-RUN yum -y install which; yum clean all
-RUN yum -y install curl; yum clean all
-RUN yum -y install nmap-ncat; yum clean all
-RUN yum -y install python python3-setuptools; yum clean all
-
-RUN yum install epel-release; yum clean all
-RUN yum update -y
-RUN yum install -y supervisor; yum clean all
-
-RUN yum -y install java-11-openjdk java-11-openjdk-devel; yum clean all
-
-ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
+RUN yum -y install epel-release \
+    && yum -y update \
+    && yum -y install \
+    curl \
+    java-11-openjdk-headless \
```

Comment:
is there a jre version for centos7 as well?

Reply:
Good question. I tried looking for one and couldn't find it. There is a `java-11-openjdk-devel` which I think is JDK, but when I install the package in the pull request, it still includes the `-devel` package. So I'm not sure.

Reply:
KK. No worries. This is a huge improvement. Thanks~